Monika After Story/game/script-topics.rpy

```diff
@@ -8511,6 +8511,238 @@ label monika_citizenship:
     m 1eka "Thank you for listening to me vent, [player]."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_idle_game",category=['games'],prompt="I'm going to game for a bit",random=False,pool=True))
```

Comment:
Moving this to a new category for brb's, also going to want this unlocked always.
```suggestion
    addEvent(Event(persistent.event_database,eventlabel="monika_idle_game",category=['be right back'],prompt="I'm going to game for a bit",random=False,pool=True,unlocked=True))
```